packages/docs/src/pages/index.md

```diff
@@ -9,14 +9,8 @@ It is currently being applied to [HealthCare.gov](https://www.healthcare.gov/).
 
 ## Getting started
 
-<div class="ds-c-alert">
-  <div class="ds-c-alert__body">
-    <h2 class="ds-c-alert__heading">CMS Design System v4 has been released</h2>
-    <p class="ds-c-alert__text">For more information on v4 and how to upgrade, see our <a href="{{root}}/startup/migrating-v4/">migration guide</a>.</p>
-  </div>
-</div>
-
-Get started by [downloading the code as a zip file](https://github.com/CMSgov/design-system/releases/latest), [installing with NPM](/startup/installation/), or [downloading the Sketch UI kit](https://github.com/CMSgov/design-system/raw/master/design-assets/CMS-Design-System-UI-kit.sketch) for wireframes and prototypes.
+- **For developers** - [Download the code as a zip file](https://github.com/CMSgov/design-system/releases/latest) or [install with NPM](/startup/installation/).
+- **For designers** - [Get the Sketch Library](sketch://add-library?url=https%3A%2F%2Fgithub.com%2FCMSgov%2Fdesign-system%2Fraw%2Fmaster%2Fdesign-assets%2Fcms-design-system-ui-kit.xml) or [downloading the Sketch UI kit](https://github.com/CMSgov/design-system/raw/master/design-assets/CMS-Design-System-UI-kit.sketch).
```

Comment:
Typo - should be `download the Sketch UI kit` not `downloading`

Noting that I do not see `Get the Sketch Library` represented as a link when I look at the preview of this PR in the github interface. Does it work locally?

![image](https://user-images.githubusercontent.com/4106356/79367328-89654900-7f02-11ea-9a21-419e2dea926e.png)

Reply:
Thanks for flagging this - it does work locally 👍